less/notifications.less

```diff
@@ -0,0 +1,33 @@
+.id7-notifications-badge {
+  font-size: 0.8em;
+  animation-name: bounce;
```

Comment:
This is fine for now but I think, on reading back in the channel history, that the preferred animation was fade, not bounce?